jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -263,9 +266,8 @@ protected ContextHandler(Context context, HandlerContainer parent, String contex
         _scontext = context == null ? new Context() : context;
         _attributes = new AttributesMap();
         _initParams = new HashMap<>();
-        addAliasCheck(new ApproveNonExistentDirectoryAliases());
         if (File.separatorChar == '/')
-            addAliasCheck(new AllowSymLinkAliasChecker());
+            addAliasCheck(new SymlinkAllowedResourceAliasChecker(this));
```

Comment:
I'm wondering if we really still need this by default?   I guesss too late to change in a dot release, but let's put a TODO in to remove if/when we go to 10.1